Lib/logging/__init__.py

```diff
@@ -1079,7 +1079,7 @@ def handleError(self, record):
             t, v, tb = sys.exc_info()
             try:
                 sys.stderr.write('--- Logging error ---\n')
-                traceback.print_exception(t, v, tb, None, sys.stderr)
+                traceback.print_exc(limit=None, file=sys.stderr)
```

Comment:
Why not `print_exception(v, limit=None, file=sys.stderr)`? That seems more in the spirit of the PR.